tasks/publish-to-cgw/publish-to-cgw.yaml

```diff
@@ -114,7 +115,9 @@ spec:
             Generate metadata for each file in
             content_list that starts with the component name
             """
-            shortURL_base = '/pub/cgw'
+            shortURL_base = '/pub/'
+            if mirrorOpenshiftPush:
```

Comment:
The PR description says:

> Change the shortUrl for Content Gateway when delivering to mirror openshift.

Which does happen here. But I would expect that the behavior would stay the same unless mirrorOpenshiftPush is set. But in fact the code now defaults to `/pub/`. Can you confirm this is right?

Also, wasn't this script supposed to be moved to the utils image? Or maybe there's a ticket for it and it will happen later? Just curious.

Reply:
Yes, this is correct.
They set up mirror.openshift.com to mirror from `/pub/cgw`
So when we don't want it mirrored, we omit the `cgw` portion. (Not super intuitive, but that's how it was set up)
In both cases, the content always goes to content-gateway.

As for the script being moved to utils image... yes.  https://issues.redhat.com/browse/CLDX-182 coming soon

Reply:
OK, thanks for confirming and for the link.